doc/learn/build-a-syntax-tree-typescript.md

```diff
@@ -0,0 +1,130 @@
+---
```

Comment:
remove `-typescript` in title if it works for both JS and TS

Reply:
removed